### PR TITLE
Feature/dividends details presenter data

### PR DESCRIPTION
**Pull request by monitz87**
**Please make sure the following boxes are checked before submitting your Pull Request:**

- [x] I've added this PR's link to its Asana task(s)
- [ ] If this PR adds new code that is not going to change in the near future, it includes unit tests to cover it.

This PR completes the dividends details screen functionality and fixes the calculations and display data

### CODE REVIEW

File: packages/new-polymath-issuer/src/pages/Home.tsx [hunk 1]
```diff
@@ -90,12 +92,24 @@ export class ContainerBase extends Component<Props> {
         name: 'My Dividend Distribution',
         amount: new BigNumber('10000'),
         checkpointId: 1,
-        excludedAddresses: [],
+        excludedAddresses: ['0x821aea9a577a9b44299b9c15c88cf3087f3b5544'],
+        // excludedAddresses: [],
```
Comment (RafaelVidaurre) [inline packages/new-polymath-issuer/src/pages/Home.tsx]: 👁

File: packages/new-polymath-issuer/src/pages/Home.tsx [hunk 2]
```diff
@@ -90,12 +92,24 @@ export class ContainerBase extends Component<Props> {
...
+      updateTaxWithholdingListStart({
+        securityTokenSymbol: 'A0T0',
+        dividendType: DividendModuleTypes.Erc20,
+        investorAddresses: ['0x0d1d4e623d10f9fba5db95830f7d3839406c6af2'],
+        percentages: [0.5],
+      })
+    );
+  };
+
   public checkForValidity = async () => {
```
Comment (RafaelVidaurre) [inline packages/new-polymath-issuer/src/pages/Home.tsx]: This log is from before this PR apparently, but lets remove it

File: packages/new-polymath-issuer/src/pages/Home.tsx (hunk 1 above)
Comment (monitz87) [reply packages/new-polymath-issuer/src/pages/Home.tsx]: Remember that this is the home screen we use to test transactions. It's not really production code so I don't see the need to keep it clean

File: packages/new-polymath-issuer/src/pages/Home.tsx (hunk 2 above)
Comment (monitz87) [reply packages/new-polymath-issuer/src/pages/Home.tsx]: Same as above

File: packages/new-polymath-issuer/src/pages/Home.tsx (hunk 1 above)
Comment (RafaelVidaurre) [reply packages/new-polymath-issuer/src/pages/Home.tsx]: Oh right, missed that. Nvm then!

**Pull request merged**